cypress/e2e/data-import/importing-marc-bib-files/updates-should-add-035-field.cy.js

```diff
@@ -200,7 +200,7 @@ describe('ui-data-import', () => {
       // upload .mrc file
       cy.visit(TopMenu.dataImportPath);
       DataImport.uploadFile('marcFileForC358998ForCreate_2.mrc', secondMarcFileNameForCreate);
-      JobProfiles.searchJobProfileForImport('Default - Create instance and SRS MARC Bib');
+      JobProfiles.searchJobProfileForImport(jobProfileToRun);
```

Comment:
You can remove some waiters to the method

Reply:
Could you clarify me what waiters do you mean?